src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.NativeAOT.targets

```diff
@@ -49,6 +49,8 @@ This file contains the NativeAOT-specific MSBuild logic for .NET for Android.
       -->
       <_OriginalSuppressTrimAnalysisWarnings>$(SuppressTrimAnalysisWarnings)</_OriginalSuppressTrimAnalysisWarnings>
       <SuppressTrimAnalysisWarnings>true</SuppressTrimAnalysisWarnings>
+      <!-- Ensure ILLink respects the value of SuppressTrimAnalysisWarnings -->
```

Comment:
This suggests that `<ILLink/>` *doesn't* currently respect `$(SuppressTrimAnalysisWarnings)`.  Is there a bug for this?

Reply:
This is happening because we are running both ILLink and ILC. We want `SuppressTrimAnalysisWarnings=false` by default so we get warnings from ILC.

But we don't want 2x trimmer warnings, so we have to turn it off and back on for ILC.  Unfortunately, `$(_ExtraTrimmerArgs)` is already set.

Same approach here:

* https://github.com/xamarin/xamarin-macios/blob/c80a942b4d31e66cc10be59024cf0ba3e5115b92/dotnet/targets/Xamarin.Shared.Sdk.targets#L676-L677